resources/dicts/events/death/general.json

```diff
@@ -1576,7 +1576,7 @@
         ],
         "tags": [ "no_body", "all_lives" ],
         "weight": 20,
-        "event_text": "m_c couldn't miss today! Such a bright day, perfect for a walk. That was until m_c detected another scent, a foul one. {PRONOUN/m_c/subject/CAP} recoiled, but not before long claws slashed across {PRONOUN/m_c/poss} face. Later, c_n found only blood and the scent of a predator at the scene.",
+        "event_text": "m_c met the long claws of a predator. Later, c_n found only {PRONOUN/m_c/poss} blood and a foul smell.",
```

Comment:
met their end might be better phrasing

Reply:
hmm counterpoint: I wanted to preserve the original "long claws slashed across their face" vibes

Reply:
"met their end to the long claws of a predator" purrhaps?

Reply:
Ooh, I like that